packages/dropdown/src/Dropdown.js

```diff
@@ -135,6 +145,19 @@ export default class Dropdown extends Component {
     return formatOption(value);
   }
 
+  static getTextEntryChangeCallback(downshiftCallback, customCallback) {
```

Comment:
This already exists in `@hig/utils` as `combineEventHandlers`

Reply:
:+1: updated in c5b6975c